### ISSUE TITLE
Use embedded playground to show samples in tutorial of "File System API"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#462]
```
Hi Eric,
I'd like to invite you to try embedded playground in the your tutorial of the 
Filesystem APIS, esp. the two examples in "Putting it all together" section.

The tutorial and help doc links are listed below:

Tutorial Link:
http://www.html5rocks.com/tutorials/file/filesystem/

Guide on how to use embedded playground:
https://docs.google.com/a/google.com/document/d/1ReJCAeh9wW-JO07Wx7uWferqg3BQwMP
dubORobNkQwU/edit?hl=en#

Feel free to let me know if you have any questions.

Thanks!

-hanrui

```

Original issue reported on code.google.com by `han...@google.com` on 25 Feb 2011 at 7:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `han...@google.com` on 25 Feb 2011 at 7:11
- Changed state: **New**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Mar 2011 at 8:43
- Added labels: **Milestone-5.5**
- Removed labels: **Milestone-5.4**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 11 Apr 2011 at 4:33
- Added labels: **Milestone-6**
- Removed labels: **Milestone-5.5**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Aug 2011 at 6:04
- Added labels: **Milestone-7.1**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision 087d64a2ae93.
Eric, could you have a look at it? Thank you!
```

Original comment by `han...@google.com` on 8 Sep 2011 at 3:36
- Changed state: **Fixed**
- Added labels: **Milestone-7.2**
- Removed labels: **Milestone-7.1**
